db/seed/example_studies/mouse_brain_1M_cell/study_info.json

```diff
@@ -0,0 +1,40 @@
+{
+  "study": {
+    "name": "10X 1.3 Million Brain Cells from E18 Mice",
+    "description": "(Based on SCP383) Cells from cortex, hippocampus and subventricular zone of two E18 mice. We would like to acknowledge and thank the data generator (10X Genomics) for proving this data which can be found here <a href='https://support.10xgenomics.com/single-cell-gene-expression/datasets/1.3.0/1M_neurons'>10X</a>",
```

Comment:
```suggestion
    "description": "(Based on SCP383) Cells from cortex, hippocampus and subventricular zone of two E18 mice. We would like to acknowledge and thank the data generator (10x Genomics) for proving this data which can be found here <a href='https://support.10xgenomics.com/single-cell-gene-expression/datasets/1.3.0/1M_neurons'>10X</a>",
```

Reply:
I'm going to keep this  since it's copied from the study in production as-is

Reply:
FWIW, I just fixed the typo in production, and smoothened the link text.